backend/apid/routers/lister_test.go

```diff
@@ -92,8 +92,8 @@ func TestList(t *testing.T) {
 			w := httptest.NewRecorder()
 
 			router := mux.NewRouter()
-			router.PathPrefix("/foo/{subcollection}").HandlerFunc(List(controller.List))
-			router.PathPrefix("/foo").HandlerFunc(List(controller.List))
+			router.PathPrefix("/foo/{subcollection}").HandlerFunc(List(controller.List, corev2.CheckConfigFields))
+			router.PathPrefix("/foo").HandlerFunc(List(controller.List, corev2.CheckConfigFields))
```

Comment:
I would suggest passing in `func(r corev2.Resource) map[string]string { return map[string]string{} }`, a "no-op" `FieldsFunc`, if you just need one for the sake of passing one in.